projects/odf_transform/test/test_headers/CTD_HUD2018030_003_01_DN.ODF.json.nc.txt

```diff
@@ -0,0 +1,1231 @@
+		QCNTR_02:original_PRINT_FIELD_WIDTH = "2" ;
+		QCNTR_02:original_PRINT_DECIMAL_PLACES = "0" ;
+		QCNTR_02:original_ANGLE_OF_SECTION = "-99.000000" ;
+		QCNTR_02:original_MAGNETIC_VARIATION = "-99.000000" ;
+		QCNTR_02:original_DEPTH = "0.000000" ;
+		QCNTR_02:original_MINIMUM_VALUE = "1" ;
+		QCNTR_02:original_MAXIMUM_VALUE = "1" ;
+		QCNTR_02:original_NUMBER_VALID = "62" ;
+		QCNTR_02:original_NUMBER_NULL = "0" ;
+		QCNTR_02:pcode = "QCNTR" ;
+		QCNTR_02:pcode_number = 2LL ;
+		QCNTR_02:long_name = "QualityFlag:CNTR_02" ;
+		QCNTR_02:null_value = -99.f ;
+	double QCFF_01(z) ;
+		QCFF_01:_FillValue = -99. ;
+		QCFF_01:original_TYPE = "DOUB" ;
+		QCFF_01:original_NAME = "Quality flag: QCFF" ;
+		QCFF_01:original_UNITS = "none" ;
+		QCFF_01:original_CODE = "QCFF_01" ;
+		QCFF_01:original_NULL_VALUE = "-99.0" ;
+		QCFF_01:original_PRINT_FIELD_WIDTH = "5" ;
+		QCFF_01:original_PRINT_DECIMAL_PLACES = "0" ;
+		QCFF_01:original_ANGLE_OF_SECTION = "0.000000" ;
+		QCFF_01:original_MAGNETIC_VARIATION = "0.000000" ;
+		QCFF_01:original_DEPTH = "0.000000" ;
+		QCFF_01:original_MINIMUM_VALUE = "0" ;
+		QCFF_01:original_MAXIMUM_VALUE = "0" ;
+		QCFF_01:original_NUMBER_VALID = "62" ;
+		QCFF_01:original_NUMBER_NULL = "0" ;
+		QCFF_01:pcode = "QCFF" ;
+		QCFF_01:pcode_number = 1LL ;
+		QCFF_01:null_value = -99. ;
+	float PRESPR01(z) ;
+		PRESPR01:_FillValue = NaNf ;
+		PRESPR01:long_name = "Pressure" ;
+		PRESPR01:standard_name = "sea_water_pressure" ;
+		PRESPR01:units = "decibar" ;
+	float TEMPS601(z) ;
+		TEMPS601:_FillValue = NaNf ;
+		TEMPS601:long_name = "Sea Water Temperature" ;
+		TEMPS601:standard_name = "sea_water_temperature" ;
+		TEMPS601:units = "deg_C" ;
+	float TEMPS602(z) ;
+		TEMPS602:_FillValue = NaNf ;
+		TEMPS602:long_name = "Sea Water Temperature" ;
+		TEMPS602:standard_name = "sea_water_temperature" ;
+		TEMPS602:units = "deg_C" ;
+	float PSALST01(z) ;
+		PSALST01:_FillValue = NaNf ;
+		PSALST01:long_name = "Sea Water Practical Salinity" ;
+		PSALST01:standard_name = "sea_water_practical_salinity" ;
+		PSALST01:units = "PSS-78" ;
+	float PSALST02(z) ;
+		PSALST02:_FillValue = NaNf ;
+		PSALST02:long_name = "Sea Water Practical Salinity" ;
+		PSALST02:standard_name = "sea_water_practical_salinity" ;
+		PSALST02:units = "PSS-78" ;
+
+// global attributes:
+		:featureType = "profile" ;
+		:summary = "brief summary of dataset" ;
+		:title = "dataset title" ;
+		:institution = "DFO BIO" ;
+		:history = "" ;
+		:infoUrl = "http://www.pac.dfo-mpo.gc.ca/science/oceans/data-donnees/index-eng.html" ;
+		string :header = "[snip]" ;
+		:description = "detailed description" ;
+		:keywords = "temperature, salinity" ;
+		:acknowledgement = "thanks etc ..." ;
+		:id = "HUD2018030-003-01" ;
+		:naming_authority = "COARDS" ;
+		:comment = "" ;
+		:creator_name = "CIOOS" ;
+		:creator_email = "cioos email here" ;
+		:creator_url = "http://www.someurlhere.com" ;
+		:license = "standard license information here" ;
+		:project = "SCOTIAN SHELF" ;
```

Comment:
Project should be ATLANTIC ZONE MONITORING PROGRAM (AZMP) SCOTIAN SHELF;
CRUISE_HEADER,
  CRUISE_NAME='SCOTIAN SHELF',
  CRUISE_DESCRIPTION='ATLANTIC ZONE MONITORING PROGRAM (AZMP)'